### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by mj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.1.0](https://github.com/voxpupuli/puppet-dhcp/tree/v5.1.0) (2023-12-11)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v5.0.1...v5.1.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#273](https://github.com/voxpupuli/puppet-dhcp/pull/273) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (kenyon) [inline CHANGELOG.md:13]: Needs to be v6.0.0 due to the breaking change.

**Review by kenyon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mj) [reply CHANGELOG.md:13]: Thanks for the feedback. I've updated the PR accordingly.

**Comment by kenyon** [comment]
We need #274 (modulesync) merged first.

File: CHANGELOG.md [hunk 2]
```diff
@@ -60,7 +73,7 @@ These should not affect the functionality of the module.
 
 ## [v4.0.0](https://github.com/voxpupuli/puppet-dhcp/tree/v4.0.0) (2020-02-24)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v3.4.0...v4.0.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/5.1.0...v4.0.0)
```
Comment (smortex) [inline CHANGELOG.md:77]: Hum, the tags dates seem to be inconsistent.  Maybe moving the old entries to HISTORY.md to generate changelog only for new entries would help?

Comment (smortex) [reply CHANGELOG.md:77]: Versions without a `v` seems to relate to `theforeman-dhcp`, while the ones with a `v` relate to `puppet-dhcp`.

When browsing a version without the `v`, GitHub warns:

> [!WARNING]
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Removing these tags may help?

Comment (kenyon) [reply CHANGELOG.md:77]: Oh yeah, that's weird. I don't think I have permission to remove the foreman tags.

Comment (kenyon) [reply CHANGELOG.md:77]: Oh, no, looks like I can delete the foreman tags from the web interface. I'll do that.

Comment (kenyon) [reply CHANGELOG.md]: @smortex how does that look now? Looks good to me.

**Review by smortex — APPROVED** [review]
That looks better!

**Pull request merged**